recipes/meta.yaml

```diff
@@ -10,13 +10,15 @@ package:
 
 source:
   git_url: https://github.com/openghg/openghg.git
-  git_rev: {{ GIT_DESCRIBE_TAG }}
+  git_rev: {{ GIT_TAG }}
   git_depth: 1
```

Comment:
I _think_ that in order for the `GIT_DESCRIBE_TAG` bit in the `version` section to work, the depth here needs to be set to more than `1`. I'd just remove this `git_depth: 1` line as it's not required field and the repo is not so large that it should cause a problem.

Reply:
Its removed and pushed to PR.